src/cmake/after_ws_install.cmake

```diff
@@ -13,3 +13,4 @@ if( WIN32 )
 endif()
 
 install(FILES "${CMAKE_1V1ARENA_DIR}/conf/1v1arena.conf.dist" DESTINATION ${CONF_DIR})
+
```

Comment:
If you added a space on purpose, it's not entirely needed if it has a carriage return at the end (\r different from a new line \n)

Reply:
My mistake , copy paste typo